app/helpers/breadcrumb_helper.rb

```diff
@@ -0,0 +1,14 @@
+module BreadcrumbHelper
+  def breadcrumb_link_to(title, url, *args)
+    content_tag(:li, title.titleize, class: 'govuk-breadcrumbs__list-item') do
```

Comment:
```suggestion
    ```suggestion
     # rubocop:disable Rails/OutputSafety
    content_tag(:li, title.titleize, class: 'govuk-breadcrumbs__list-item') do
      link_to(title.titleize, url, *args, class: 'govuk-breadcrumbs__link')
    end.html_safe
    # rubocop:enable Rails/OutputSafety
```
```